pkg/agent/utils.go

```diff
@@ -850,7 +850,7 @@ func IsDynamicKubeletEnabled(cs *datamodel.ContainerService, dynamicKubeletToggl
 
 func ensureKubeletConfigFlagsValue(kc map[string]string, cs *api.ContainerService, dynamicKubeletToggleEnabled bool) {
```

Comment:
could just change the signature of this func to take *datamodel.ContainerService

Reply:
Yes, I'll make the change.

Reply:
Code updated.